code-challenges.test.js

```diff
@@ -28,88 +28,160 @@
+    // Expected output: ["Temperature", "Database", "Chopsticks", "Mango", "Deduction"]
+    expect(capitalized(randomNouns1)).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]),
+    expect(capitalized(randomNouns2)).toEqual(["Temperature", "Database", "Chopsticks", "Mango", "Deduction"])
   })
 })
 
+// FAIL  ./code-challenges.test.js
+//  capitalized
+//    ✕ returns an array with all the words capitalized
+//
+//  ● capitalized › returns an array with all the words capitalized
+//
+//    ReferenceError: capitalized is not defined
+
+// Pseudocode
+// Create a function called capitalized
+// input - array datatype
+// use built in method toUpperCase()
+// Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
+// Expected output: ["Temperature", "Database", "Chopsticks", "Mango", "Deduction"]
+
 // b) Create the function that makes the test pass.
 
+const capitalized = (array) => {
+  return array.map(value => value[0].toUpperCase() + value.substring(1))
+}
+console.log(capitalized(randomNouns1))
+console.log(capitalized(randomNouns2))
 
+// PASS  ./code-challenges.test.js
+//  capitalized
+//    ✓ returns an array with all the words capitalized (1 ms)
 
 // --------------------3) Create a function that takes in a string and logs the index of the first vowel.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-//const vowelTester1 = "learn"
+// const vowelTester1 = "learn"
 // Expected output: 1
-//const vowelTester2 = "academy"
+// const vowelTester2 = "academy"
 // Expected output: 0
-//const vowelTester3 = "challenges"
+// const vowelTester3 = "challenges"
 // Expected output: 2
 
-describe("vowel", () => {
-  const vowelTester1 = "learn"
-  const vowelTester2 = "academy"
-  const vowelTester3 = "challenges"
+describe("firstVowel", () => {
   it("takes in a string and logs the index of the first vowel", () => {
-    expect(vowel(vowelTester1)).toEqual(1)
-    expect(vowel(vowelTester2)).toEqual(0)
-    expect(vowel(vowelTester3)).toEqual(2)
+    const vowelTester1 = "learn"
+    // Expected output: 1
+    const vowelTester2 = "academy"
+    // Expected output: 0
+    const vowelTester3 = "challenges"
+    // Expected output: 2
+    expect(firstVowel(vowelTester1)).toEqual(1),
+    expect(firstVowel(vowelTester2)).toEqual(0)
+    expect(firstVowel(vowelTester3)).toEqual(2)
   })
 })
 
+// FAIL  ./code-challenges.test.js
+//  firstVowel
+//    ✕ returns the index of the first vowel. (1 ms)
+//
+//  ● firstVowel › returns the index of the first vowel.
+//
+//    ReferenceError: firstVowel is not defined
+
+// Pseudocode
+// Create a function called firstVowel
+// input - string datatype
+// Use for loop to evaluate first vowel.
+// Expected output: 1
+// Expected output: 0
+// Expected output: 2
+
 // b) Create the function that makes the test pass.
+
+const firstVowel = (string) => {
+  let vowelFinder = 'aeiou,AEIOU'
```

Comment:
No need for the comma inside this string.  Otherwise, excellent job on this function!